### DISCUSSION TITLE
Spotify Auth Example

### DISCUSSION THREAD

**Discussion by maxmandia** [invertase/react-native-firebase#6383] [Ideas]
Hi all,

Looking for an example using react-native-firebase auth and spotify using the Authorization Code Flow. If this already exists I apologize but would be so nice to have!

**Comment by mikehardy** [comment]
Hi there! I'm not aware of any examples doing this. The only way I can think of is to use the oath flow here along with https://github.com/FormidableLabs/react-native-app-auth/ to do the spotify login (they support it)

I don't believe there is a good example (and it may in fact need a PR here) of doing a generic Oauth integration between react-native-app-auth and `@react-native-firebase/auth` unfortunately
